src/main/java/org/jabref/logic/importer/fileformat/MarcXmlParser.java

```diff
@@ -133,8 +136,13 @@ private BibEntry parseEntry(Element element) {
          *  pages
          *  volume and number correct
          *  series and journals stored in different tags
-         *  thesis
-         *  proceedings
+         *  thesis //doctor & masterarbeit (Unterschied?) | type
+         *  proceedings //konferenzbeiträge
+         *  BibTex --> entryType
```

Comment:
```suggestion
         *  BibTeX --> entryType
```

Also: Try to write English comments.

Reply:
Absolutely, I just made a quick note of them for myself and forgot to delete them...
In the future, I will formulate them neatly right away.